### PR TITLE
Update prettier to newest version

### PR DESCRIPTION
**Pull request by MarcinJuraszek**
## Description

Update prettier to newest version

### Main changes in the PR:

1. Update prettier to newest version and auto-format the code with it

### Validation performed:

1. it builds!

### Unit Tests added:

No

### End-to-end tests added:

No

### Change file added:

Yes

### CODE REVIEW

File: change/@microsoft-teams-js-80efad11-6839-4001-9141-36de9001b0d0.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update prettier to newest version",
```
Comment (erinha) [inline change/@microsoft-teams-js-80efad11-6839-4001-9141-36de9001b0d0.json:3]: Please use past tense ("Updated...")

**Comment by erinha** [comment]
I noticed that no test files have been updated (under packages/teams-js/test). When I open the repo in VS Code in your branch and open a test file, I see that VS Code has red squiggles highlighting additional changes that are required in the test files. Is it possible for you to update the test files too?

**Review by erinha — APPROVED** [review]
I love this, thank you!

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**